### PR TITLE
add v0.5.0 change_log

### PR DESCRIPTION
**Pull request by tuofeilunhifi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,14 @@ Please refer to [data_hub.md](https://github.com/alibaba/EasyCV/blob/master/docs
 
 ## ChangeLog
 
+* 28/07/2022 EasyCV v0.5.0 was released.
+    * Self-Supervised support ConvMAE algorithm
+    * Classification support EfficientFormer algorithm
+    * Detection support FCOS、DETR、DAB-DETR and DN-DETR algorithm
+    * Segmentation support UperNet algorithm
+    * Support use torchacc to speed up training
```
Comment (Cathy0908) [inline README.md:91]: add torchacc url: https://github.com/alibaba/EasyCV/blob/master/docs/source/tutorials/torchacc.md

File: README_zh-CN.md [hunk 2]
```diff
@@ -70,6 +70,14 @@ EasyCV是一个涵盖多个领域的基于Pytorch的计算机视觉工具箱，
 
 ## 变更日志
 
+* 28/07/2022 EasyCV v0.5.0 版本发布。
+    * 自监督学习增加了ConvMAE算法
+    * 图像分类增加EfficientFormer
+    * 目标检测增加FCOS、DETR、DAB-DETR和DN-DETR算法
+    * 语义分割增加了UperNet算法
+    * 支持使用torchacc加快训练速度
```
Comment (Cathy0908) [inline README_zh-CN.md:78]: add torchacc url: https://github.com/alibaba/EasyCV/blob/master/docs/source/tutorials/torchacc.md

File: docs/source/change_log.md [hunk 3]
```diff
@@ -68,3 +68,24 @@ EasyCV support multi-gpu and multi worker training. EasyCV use DALI to accelerat
...
+# v 0.5.0 (28/07/2022)
+
+## Highlights
+- Self-Supervised support ConvMAE algorithm (([#101](https://github.com/alibaba/EasyCV/pull/101)) ([#121](https://github.com/alibaba/EasyCV/pull/121)))
+- Classification support EfficientFormer algorithm ([#128](https://github.com/alibaba/EasyCV/pull/128))
+- Detection support FCOS、DETR、DAB-DETR and DN-DETR algorithm (([#100](https://github.com/alibaba/EasyCV/pull/100)) ([#104](https://github.com/alibaba/EasyCV/pull/104)) ([#119](https://github.com/alibaba/EasyCV/pull/119)))
+- Segmentation support UperNet algorithm ([#118](https://github.com/alibaba/EasyCV/pull/118))
+
+## New Features
+- Support use torchacc to speed up training ([#105](https://github.com/alibaba/EasyCV/pull/105))
```
Comment (Cathy0908) [inline docs/source/change_log.md:81]: move torchacc to Highlights

File: README.md (hunk 1 above)
Comment (tuofeilunhifi) [reply README.md]: done

File: README_zh-CN.md (hunk 2 above)
Comment (tuofeilunhifi) [reply README_zh-CN.md]: done

File: docs/source/change_log.md (hunk 3 above)
Comment (tuofeilunhifi) [reply docs/source/change_log.md]: done

File: README.md [hunk 4]
```diff
@@ -83,6 +83,14 @@ Please refer to [data_hub.md](https://github.com/alibaba/EasyCV/blob/master/docs
 
 ## ChangeLog
 
+* 28/07/2022 EasyCV v0.5.0 was released.
+    * Self-Supervised support ConvMAE algorithm
+    * Classification support EfficientFormer algorithm
+    * Detection support FCOS、DETR、DAB-DETR and DN-DETR algorithm
+    * Segmentation support UperNet algorithm
+    * Support use [torchacc](https://github.com/alibaba/EasyCV/blob/master/docs/source/tutorials/torchacc.md) to speed up training
```
Comment (wenmengzhou) [inline README.md:91]: update tutorial list

Comment (tuofeilunhifi) [reply README.md:91]: done

**Review by Cathy0908 — APPROVED** [review]

**Pull request merged**